feeds/crates/crates.go

```diff
@@ -63,8 +72,16 @@ func (feed Feed) Latest(cutoff time.Time) ([]*feeds.Package, error) {
 			Name:        pkg.Name,
 			CreatedDate: pkg.UpdatedAt,
 			Version:     pkg.NewestVersion,
-			Type:        FeedName,
+			Type:        feed.GetName(),
 		})
 	}
 	return pkgs, nil
 }
+
+func (feed Feed) GetName() string {
+	parts := []string{FeedType}
```

Comment:
As each feed has to implement this interface & as it stands (afaict?) each is implemented in the same way, is it worth extracting the logic out into a single method?